### ISSUE TITLE
[Feature] allow for admins to set margin of error between messages

### ISSUE THREAD

**Issue by Pagwin2** [GalaxySH/cr-counting-bot#22]
this would be done to prevent a mistake from occuring when 2 people send a message at almost the same time if not the same time command could be `c?margin 5s`

**Comment by GalaxySH** [comment]
What should it do if somebody sent a count within the set margin

**Comment by nebneb0703** [comment]
I believe the point of the game is to have mistakes like that :p

**Comment by Pagwin2** [comment]
> What should it do if somebody sent a count within the set margin

delete the second message or maybe say "while one of you said the wrong number it was within the margin of error set so I'll allow it this time"

> I believe the point of the game is to have mistakes like that :p

yeah using this option should disqualify you from the multi-guild features like the custom increment and fail prevention bits

**Comment by GalaxySH** [comment]
Among other features, I have added a feature that prevents the count from being ruined if users count within a threshold of 200ms from the last count. This feature does not have an accompanying `c?margin` command, I can add one if necessary (respond with the request).

I did not add a command to set the margin because I made the feature to prevent mistakes that probably couldn't be avoided. Being able to set the margin to something like five seconds could be a future customizing option, but honestly, I think that the user deserves to fail the count if they can't realize that someone else sent the number within five seconds. I was also a bit lazy and just didn't want to have to make it customizable. Making it configurable would:

1. Force me to add that data to the database
2. Force me to make the program pull more information from the database every count
3. Keep track of message times and thresholds in the database
